packages/reports/addon/components/navi-column-config.js

```diff
@@ -115,7 +114,7 @@ class NaviColumnConfig extends Component {
         });
     }
 
-    return null;
+    return columns.length === 1 ? columns[0] : null;
```

Comment:
```suggestion
    return columns.length === 1 && column.type === 'timeDimension' ? columns[0] : null;
```
just to cover the unlikely case where you enter a report with no time grain and one column